response_operations_ui/templates/confirm-enrolment-change.html

```diff
@@ -50,8 +50,8 @@ <h1 class="jupiter">
         </dd>
       </dl>
 
-      <button type="submit" id="confirm-btn" class="btn btn--primary u-mt-s">
-        {% if change_flag == 'DISABLED' %}Disable{% else %}Enable{% endif %} enrolment
+      <button type="submit" id="confirm-btn" class="btn u-mb-s">
+        {% if change_flag == 'DISABLED' %}Disable{% else %}Enable{% endif %} enrolment?
```

Comment:
Tiny thing but there's a question mark that's now appeared on the button to disable an account - different to all other buttons.

Reply:
Addressing in next commit.